apps/docs/app/routes/index.tsx

```diff
@@ -1,8 +1,16 @@
-import { Heading, Stack } from "@vygruppen/spor-react";
+import { Box } from "@vygruppen/spor-react";
+import { ActionLinks } from "~/features/routes/index/ActionLinks";
+import { Footer } from "~/features/routes/index/Footer";
+import { HeroSection } from "~/features/routes/index/HeroSection";
+
 export default function Index() {
   return (
-    <Stack>
-      <Heading textStyle="xl-display">Welcome to us!</Heading>
-    </Stack>
+    <Box backgroundColor="alias.lightGrey" flex="1">
```

Comment:
I'm keeping the routes pretty small, so they can focus on data loading and orchestration.